tests/acceptance/features/apiSharingNg1/sharedByMe.feature

```diff
@@ -3865,7 +3865,7 @@ Feature: resources shared by user
       """
 
   @env-config
-  Scenario Outline: sharer lists the shares shared with Secure Viewer after the role is disabled (Personal Space)
+  Scenario Outline: user list the shares shared with Secure Viewer after the role is disabled (Personal Space)
```

Comment:
i think its best if we use the words sharer and sharee in case of shares

Reply:
users refers to both sharer and sharee